app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class User < ApplicationRecord
```

Comment:
Remember that a `user` has many `comments`. We should be able to obtain all of the comments made by an user by executing: `user.comments`; however, without this association we won't be able to do that:

```
irb(main):002:0> u = User.first
  User Load (0.7ms)  SELECT  "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> #<User id: 1, username: "adriaanbd", email: "adriaanbd@gmail.com", created_at: "2019-10-01 00:27:30", updated_at: "2019-10-01 00:27:30", avatar_file_name: nil, avatar_content_type: nil, avatar_file_size: nil, avatar_updated_at: nil>
irb(main):003:0> u.comments
Traceback (most recent call last):
        1: from (irb):3
NoMethodError (undefined method `comments' for #<User:0x000055a45e3c3c48>)
```
[`REQUIRED`]